spec/save_inventory/graph_of_inventory_collections_spec.rb

```diff
@@ -297,25 +300,8 @@
             init_stack_data_with_stack_stack_cycle
             init_resource_data
 
-            add_data_to_inventory_collection(@data[:orchestration_stacks],
```

Comment:
data are the same in all specs, so `@persister.<collection>.build` was moved directly to methods `init_stack_data_with_stack_stack_cycle`, `init_resource_data` etc.